content/en/docs/setup/install/multicluster/single-network/primary/index.md

```diff
@@ -0,0 +1,178 @@
+            must be on the same network, and have the same network ID.</td>
+            <td><code>${NETWORK_1}</code></td>
+        </tr>
+        <tr>
+            <td><code>meshID</code></td>
+            <td>Specifies a mesh ID as an arbitrary string. All clusters in your mesh share
+            the same mesh ID.</td>
+            <td><code>${MESH}</code></td>
+        </tr>
+        <tr>
+            <td><code>meshExpansion</code></td>
+            <td>[Recommended] Exposes the Istio control plane through the ingress gateway
+            of the mesh. Enable this option to connect remote clusters to a control
+            plane.</td>
+            <td><code>enabled: true</code></td>
+        </tr>
+    </tbody>
+</table>
+
+Using the previously [set environment variables,](/docs/setup/install/multicluster/#env-var)
+deploy an Istio control plane with the following steps:
+
+1. To pass configuration values to the Istio operator for installation,
+   define a custom resource (CR). Define and save the `install.yaml` CR with
+   the following command:
+
+{{< text bash >}}
+$ cat <<EOF> ${WORK_DIR}/${CLUSTER_2}/install.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: ${MESH}
+      multiCluster:
+        clusterName: ${CLUSTER_2}
+      network: ${NETWORK_1}
+
+      meshNetworks:
+        ${NETWORK_1}:
+          endpoints:
+          # Always use ‘Kubernetes' as the registry name for
+          # endpoints that reside on the same network as this
+          # instance of the Istio control plane.
+          - fromRegistry: Kubernetes
+          gateways:
+          - registry_service_name: istio-ingressgateway.istio-system.svc.cluster.local
+            port: 443
+      # Expose the control plane through istio-ingressgateway.
+      meshExpansion:
+        enabled: true
+EOF
+{{< /text >}}
+
+1. Install Istio on Cluster-2 with the following command:
+
+    {{< text bash >}}
+    $ istioctl --context=${CTX_2} manifest apply -f \
+    ${WORK_DIR}/${CLUSTER_2}/install.yaml
+    {{< /text >}}
+
+1. Verify that the control plane is ready with the following command:
+
+    {{< text bash >}}
+    $ kubectl --context=${CTX_2} -n istio-system get pod
+    NAME                                    READY   STATUS    RESTARTS   AGE
+    istiod-f756bbfc4-thkmk                  1/1     Running   0          136m
+    prometheus-b54c6f66b-q8hbt              2/2     Running   0          136m
+    {{< /text >}}
+
+1. After the status of all pods is `Running`, you can continue configuring
+   your deployment.
+
+**Congratulations!**
+
+You successfully added a control plane to your primary
+cluster.
+
+Next, configure endpoint discovery to support cross-cluster load balancing.
+
+## Configure endpoint discovery
+
+To enable cross-cluster load balancing in your mesh, configure endpoint
+discovery. This feature requires that clusters share secrets between them. If
```

Comment:
What secrets and for what purpose

Reply:
And are they rotated?

Reply:
These are the secrets for accessing the remote API servers (they don't expire). We should make that more clear.  Maybe:

```
In order to enable endpoints to be discovered by control planes in other clusters,
we need to share secrets that allow each control plane to access the
Kubernetes API Server in each cluster. This allows each control plane
to discover all of the endpoints the mesh.
```

Reply:
This procedure doesn't require full understanding of secrets and we provide steps to generate and push the secrets to a cluster. A PR introducing better security conceptual content about secrets would allow us to provide a link for readers to learn more.